src/index.ts

```diff
@@ -67,26 +68,28 @@ const Supergood = () => {
       config,
       metadata,
       tags,
-      isWithinContext,
+      trace,
```

Comment:
What's the use case for initializing the client with an essentially global trace? Serverless maybe? Or for if no trace defined?

Reply:
its mainly to support the `startCapture` and `withCapture`